include/sway/tree/view.h

```diff
@@ -69,10 +69,8 @@ enum sway_view_prop {
 struct sway_view_impl {
 	const char *(*get_prop)(struct sway_view *view,
 			enum sway_view_prop prop);
-	void (*set_size)(struct sway_view *view,
-			int width, int height);
-	void (*set_position)(struct sway_view *view,
-			double ox, double oy);
+	void (*configure)(struct sway_view *view, double ox, double oy, int width,
+		int height);
```

Comment:
I wonder if we should be passing layout coords to this

Reply:
I think so. It's only used for Xwayland and floating, both need layout coords.

Reply:
Since `configure` is also responsible for updating the container view, I'm not sure about this change yet